### DISCUSSION TITLE
Average Loss for traffic dataset with TempFlowEstimator

### DISCUSSION THREAD

**Discussion by sege01** [zalandoresearch/pytorch-ts#71] [Q&A] [answered]
Hello,

I am getting a very high average loss value while training the Traffic dataset with TempFlowEstimator. 
@kashif Could you please help in understanding the reason behind it and what could be done about it?
Please find the attached screenshot.
![TrafficDataset](https://user-images.githubusercontent.com/85530006/129370008-06c00de2-9895-475b-8dd6-676fd66e46dd.JPG)

Thank you!

**Comment by sege01** [comment, marked as answer]
The loss looks fine after setting scaling=False.

Thanks!!

**Reply by kashif** [reply]
@sege01 right... since traffic's domain is in (0,1) there is no need for scaling in this case.
